ibc_data/ibc_conditions.tsv

```diff
@@ -0,0 +1,380 @@
+PreferenceFaces	face_constant	Classify the level of pleasantness of a human face displayed on the screen in terms of willingness to meet the person portrayed, this condition serves as an occurrence regressor when formulated as visual evaluation of an item vs. fixation
+PreferenceFaces	face_linear	Classify the level of pleasantness of a human face displayed on the screen in terms of willingness to meet the person portrayed. this condition captures the linear effect of pleasantness (akin to judgement effects) when formulated as visual preference vs. no preference
+PreferenceFaces	face_quadratic	Classify the level of pleasantness of a human face displayed on the screen in terms of willingness to meet the person portrayed. this condition captures the quadratic effect of pleasantness (akin to confidence effects) when formulated as  confidence in preference vs. no confidence
+PreferenceHouses	house_constant	Classify the level of pleasantness of a house displayed on the screen in terms of willingness to live in that house. this condition serves as an occurrenceregressor when formulated as visual evaluation of an item vs. fixation
+PreferenceHouses	house_linear	Classify the level of pleasantness of a house displayed on the screen in terms of willingness to live in that house. this condition captures the linear effect of pleasantness (akin to judgement effects) when formulated as visual preference vs. no preference
+PreferenceHouses	house_quadratic	Classify the level of pleasantness of a house displayed on the screen in terms of willingness to live in that house. this condition captures the quadratic effect of pleasantness (akin to confidence effects) when formulated as  confidence in preference vs. no confidence
+TheoryOfMind	belief	Read a false-belief story
+TheoryOfMind	photo	Read a false-photograph story
+EmotionalPain	physical_pain	Read story about fictional characters suffering from physical pain
+EmotionalPain	emotional_pain	Read story about fictional characters suffering from emotional pain
+PainMovie	movie_pain	Watch movie-scene wherein characters experience physical pain
+PainMovie	movie_mental	Watch movie-scene wherein characters experience changes in beliefs, desires, and/or emotions
+VSTM	vstm_linear	Judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, response to constant numerosity vs. fixation
+VSTM	vstm_constant	Judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, response to numerosity vs. fixation
+VSTM	vstm_quadratic	Judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, response to numerosity interaction vs. fixation

```

Comment:
This is another one I think we can look carefully at, because in the documentation we only have the following: 
```
judge whether any bar changed orientation within two consecutive displays of bar sets on the screen
```
and then in another table we have `constant` , `linear` and `cuadratic`.

Maybe in these cases we can leave it as it is (for the tex files), as long as all the information is there.

Reply:
Some notes (this is not about the csv files but about the documentation tex file):

In the tex file, the only condition is described as follows:

> _vstm response to constant numerosity_ judge whether any bar changed orientation within two consecutive displays of bar sets on the screen

And after, the contrast are described as follows:

> _vstm constant_  VSTM response to constant numerosity vs. fixation

I think this might be a bit unclear, and also the condition description is neglecting the change of numerosity in the bars. So maybe we could complement it with something like this:

> _vstm response to numerosity_ judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, the number of bars might vary from 1 to 6 bars.

And the contrast: 

>  _vstm constant_  VSTM response to constant numerosity effect (constant numerosity vs. fixation)

and so on...

Reply:
Indeed !